rules/Proj0023.md

```diff
@@ -0,0 +1,45 @@
+# Proj0023: Use forward slashes in paths
+The use of forward slashes (`/`) is preferred as they work both for UNIX and
```

Comment:
Are there any downfalls, exceptions to using forward slashed? (3rd party packages, backwards compatibility) And should that be mentioned in the rule?

Reply:
The only downfall I can think of, is that Visual Studio generated paths to projects with `\`, but I do not like to mention that. What do you think?

Reply:
Hmm, no I don't think that needs to be mentioned